### PR TITLE
Support iso language codes and fail on invalid language

### PR DESCRIPTION
**Pull request by tmbo**
Proposed changes:
- fail if language is passed, which is not supported by underlying version of duckling
- accept iso language codes (e.g. `"en"`) as well as `"en$core"`

### CODE REVIEW

File: duckling/language.py [hunk 1]
```diff
@@ -20,3 +20,41 @@ class Language(object):
     UKRAINIAN = 'uk$core'
     VIETNAMESE = 'vi$core'
     CHINESE = 'zh$core'
+
+    SUPPORTED_LANGUAGES = [
```
Comment (FraBle) [inline duckling/language.py]: You could make this a set for a faster lookup (even though the difference should be minor).
Other than that it looks good! What do you think?

Comment (tmbo) [inline duckling/language.py]: sure thing, done.

**Comment by FraBle** [comment]
Merged and released as [v1.3.1](https://pypi.python.org/pypi?:action=display&name=duckling&version=1.3.1) :)

**Comment by tmbo** [comment]
Great, thanks a lot :)

**Pull request merged**